### PR TITLE
Add Faker.App.semver

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/faker/app.ex [hunk 1]
```diff
@@ -19,13 +19,41 @@ defmodule Faker.App do
...
+
+  * `:allow_pre` - when `true`, a pre-release identifier (e.g.: `-dev`)
+    will be appended (default: `true`)
+  * `:allow_build` - when `true`, a build identifier (e.g.: `+001`)
+    will be appended (default: `true`)
+  """
+  @spec semver(Keyword.t) :: String.t
+  def semver(opts \\ []) do
+    allow_pre = Keyword.get(opts, :allow_pre, true)
+    allow_build = Keyword.get(opts, :allow_build, true)
```
Comment (igas) [inline lib/faker/app.ex]: I think both defaults should be false, it's more common to have 1.2.3 format.

Comment (wojtekmach) [inline lib/faker/app.ex]: yup, I was on a fence and ended up using a default that will show people a broader range of values, but I guess you're right that's it's more common the other way around.

Fixed in 5d81e85915da2fe27a84548ac35edfdc2b03b414

PS thanks for this project!

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: lib/faker/app.ex (hunk 1 above)
Comment (igas) [inline lib/faker/app.ex]: Thanks for contribution!

**Pull request merged**